extensions/wikia/CommunityPage/styles/components/_help.scss

```diff
@@ -6,23 +6,26 @@
 $small-icon-size: 18px;
 $big-icon-size: 30px;
 $help-module-margin: 12px;
-$help-module-padding-bottom: 20px;
+$help-module-padding: 24px;
+$help-module-padding-side: 18px;
 $help-module-link-padding: 12px;
+$help-module-title-padding: 3px;
 $policy-link-padding-top: 18px;
 $policy-link-margin-top: 14px;
```

Comment:
@lukaszkonieczny that's exactly the case when we can use it - `$help-module-spacing: 12px;` and then:  `.community-page-help-module-title {
  		margin-left: $help-module-spacing;` and `&.community-page-module-box {
 +		padding: $help-module-margin * 2 $help-module-padding-side;